Dockerfile-alpine.template

```diff
@@ -70,6 +69,8 @@ RUN set -eux; \
 	./configure \
 		--build="$gnuArch" \
 		--without-jemalloc \
+		--with-rst2man=$(command -v true) \
```

Comment:
`$(command -v true)` is pointless, but just to avoid confusion of what it means here...